pandas/tests/io/test_parquet.py

```diff
@@ -252,6 +301,10 @@ def test_duplicate_columns(self, fp):
 
     def test_unsupported(self, fp):
 
+        # bool with none
+        df = pd.DataFrame({'a': [True, None, False]})
```

Comment:
We're supporting the pandas 0.x semantics as is in pyarrow

```
In [13]: import pyarrow as pa

In [14]: import pandas as pd

In [15]: df = pd.DataFrame({'a': [True, None, False]})

In [16]: t = pa.Table.from_pandas(df)

In [17]: t
Out[17]: 
pyarrow.Table
a: bool

In [19]: t[0].to_pandas()
Out[19]: 
0     True
1     None
2    False
Name: a, dtype: object
```

In pandas2, boolean will probably be a bitarray with a separate null bitmap, but in the meantime we're stuck with this =(